packages/expo-cli/src/update.ts

```diff
@@ -1,16 +1,37 @@
-import { ModuleVersion } from '@expo/xdl';
+import * as PackageManager from '@expo/package-manager';
+import boxen from 'boxen';
+import checkForUpdate from 'update-check';
 
+import log from './log';
+
+// We use require() to exclude package.json from TypeScript's analysis since it lives outside the
+// src directory and would change the directory structure of the emitted files under the build
+// directory
 const packageJSON = require('../package.json');
 
-const ModuleVersionChecker = ModuleVersion.createModuleVersionChecker(
-  packageJSON.name,
-  packageJSON.version
-);
+export default async function shouldUpdate(): Promise<void> {
+  const update = checkForUpdate(packageJSON, { interval: 1 }).catch(() => null);
```

Comment:
the interval is in ms here, so this means we wouldn't cache the update at all. if we leave out the interval option it should default to one day, which is what we had before.